src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+        getCommand();
     }
-}
+
+    private static void getCommand() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        // get user input
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                showList();
+                line = in.nextLine();
+                continue;
+            }
+            if (line.startsWith("done")) {
+                int number = Integer.parseInt(line.substring(line.length() - 1));
+                markAsDone(number);
+                line = in.nextLine();
+                continue;
+            }
+            // Command is valid, handle the command
+            System.out.println("    ____________________________________________________________");
+            System.out.println("     Got it. I've added this task: ");
+            handleCommand(line);
+            System.out.println("     Now you have " + descriptions.size() + " tasks in the list.");
+            System.out.println("    ____________________________________________________________");
+            line = in.nextLine();
+        }
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Bye. Hope to see you again soon!");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    private static void handleCommand(String line) {
+        String description;
+        String date;
+        dones.add(false);
+        if (line.startsWith("todo")) {
+            types.add("T");
+            description = line.substring(5);
+            Todo todo = new Todo(description);
+            descriptions.add(description);
+            dates.add("");
+            System.out.println("       " + todo.toString(false));
+        } else if (line.startsWith("deadline")) {
+            types.add("D");
+            int separator = line.indexOf("/by");
+            description = line.substring(9, separator - 1);
+            date = line.substring(separator + 4);
+            Deadline deadline = new Deadline(description, date);
+            descriptions.add(description);
+            dates.add(date);
+            System.out.println("       " + deadline.toString(false));
+        } else if (line.startsWith("event")) {
+            types.add("E");
+            int separator = line.indexOf("/at");
+            description = line.substring(6, separator - 1);
+            date = line.substring(separator + 4);
+            Event event = new Event(description, date);
+            descriptions.add(description);
+            dates.add(date);
+            System.out.println("       " + event.toString(false));
+        } else {
+            types.add("unknown");
+        }
+    }
+
+    private static void showList() {
```

Comment:
I like methods like this where the name makes the intention obvious, would love to see more of such methods.